lib/internal/validators.js

```diff
@@ -258,6 +258,21 @@ function validateUnion(value, name, union) {
   }
 }
 
+function validateLinkHeaderValue(value, name) {
+  const linkValueRegExp = /((<([^>]*)>;)\s*(rel="?([^;]*)"?;)\s*(as="?([^\s]*)"?))/;
```

Comment:
Shouldn't we be anchoring this regexp so that it matches the string exactly?

Reply:
updated, also ran some tests here 👉 https://regexr.com/6rjub